packages/tsconfig/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@4c/tsconfig",
+  "version": "0.1.0",
+  "main": "base.json",
```

Comment:
@jquense please double check these values to make sure they are actually correct 🙏

Reply:
yup looks fine